lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -910,8 +915,9 @@ def _check_template_cube(cube):
         raise ValueError(msg)
 
     @staticmethod
-    def _check_unit_compatibility(mean_values, variance_values,
-                                  probability_cube_template):
+    def _check_unit_compatibility(
+            location_parameter_values, scale_parameter_values,
+            probability_cube_template):
         """
         The mean, variance, and threshold values come from three different
```

Comment:
Docsting needs updating too

Reply:
I've updated this docstring, so hopefully it is now a bit clearer.